quintilesims/auth0-proxy
# Update for Layer0 v0.10.4

The `--wait` flag was removed in Layer0 v0.10.4, and so the same change needed to be made in this project. Additionally, I've created a README that explains how to use the Auth0 Proxy.

**How to test**
Create a terraform deployment that uses the Auth0 Proxy module and also has a service deployed behind a private load balancer. Confirm that the service works properly when using the proxy's load balancer URL.

Notes: There is an `auth0 proxy test` Auth0 client that I've been using; you'll want to add the Auth0 Proxy's load balancer to the list of allowed callback URLs on the Auth0 client.

Closes #1 
Closes #2
